openvdb_points/tools/AttributeArray.h

```diff
@@ -1506,70 +1582,88 @@ AttributeHandle<T>::AttributeHandle(const AttributeArray& array, const bool pres
     mFiller = typedAccessor->mFiller;
 }
 
+template <typename T, bool Strided, bool Interleaved>
+Index AttributeHandle<T, Strided, Interleaved>::index(Index n, Index m) const
+{
+    if (Strided) {
+        if (Interleaved)    return m * size() + n;
+        else                return n * stride() + m;
```

Comment:
This seems to be the only use case of the Strided and Interleaved templated args. On the creation of the attribute handle, could these not be bools which are set from array.isStrided() array.isInterleaved()?